contracts/rate_oracles/BaseRateOracle.sol

```diff
@@ -12,9 +12,8 @@ import "../core_libraries/Time.sol";
 import "../utils/WadRayMath.sol";
 
 /// @notice Common contract base for a Rate Oracle implementation.
-///  This contract is abstract. To make the contract deployable override the
-/// `getLastUpdatedRate` function and the `UNDERLYING_YIELD_BEARING_PROTOCOL_ID` constant.
-/// @dev Each specific rate oracle implementation will need to implement the virtual functions
+///  This contract is abstract and partially implemented by the Compounding and Linear Contracts.
```

Comment:
maybe "This contract is abstract that's inherited by both Compounding and Linear Rate Oracles"?

Reply:
what about now? [c2a5c2f](https://github.com/Voltz-Protocol/voltz-core/pull/230/commits/c2a5c2fda8ffd7f8ede64ea6b3b1d51a822d32d0)